crates/telio-nurse/src/qos.rs

```diff
@@ -37,7 +39,7 @@ pub struct NodeInfo {
     pub connected_time: Duration,
 
     // RTT
-    pub endpoint: DualTarget,
+    pub ip_address: DualTarget,
```

Comment:
At this point would it not make more sense to simply accept a Vec<IpAddr> here, and in pinging logic just test all until at least one of it responds ?

Reply:
done